flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitorITCase.java

```diff
@@ -262,8 +247,8 @@ public void testRedirectToLeader() throws Exception {
 				response = followingClient.getNextResponse(deadline.timeLeft());
 				assertEquals(HttpResponseStatus.OK, response.getStatus());
 				assertEquals(response.getType(), MimeTypes.getMimeTypeForExtension("json"));
-				assertTrue(response.getContent().contains("\"taskmanagers\":1") ||
-						response.getContent().contains("\"taskmanagers\":0"));
+				assertThat(response.getContent(), either(containsString("\"taskmanagers\":1"))
```

Comment:
I've replaced assertTrue/False because assertThat and hamcrest matchers provide much better error messages. Instead of weird 'java.lang.AssertionError' we can get a more readable message in the log without effort.
Of cource that's a matter of opinion and I can revert it. Should I do?